cmd/storage-rest-server.go

```diff
@@ -47,6 +48,23 @@ func (s *storageRESTServer) writeErrorResponse(w http.ResponseWriter, err error)
 	w.(http.Flusher).Flush()
 }
 
+type bulkErrorsResponse struct {
+	Errs []error `json:"errors"`
+}
+
+func (s *storageRESTServer) writeErrorsResponse(w http.ResponseWriter, errs []error) {
+	w.WriteHeader(http.StatusOK)
```

Comment:
This is redundant writes over `w` will automatically write this.

Reply:
> This is redundant writes over `w` will automatically write this.

Done